templateutils/templateutils.go

```diff
@@ -63,8 +64,27 @@ Some new functions have been added to Go's template language
   field for each struct on a new line , e.g.,
 
   {{select . "Name"}}
+- table outputs a table given an array or a slice of structs.  The table headings are taken
+  from the names of the structs fields.  Hidden fields and fields of type channel are ignored.
```

Comment:
> The table headings are taken from the names of the structs fields

Would it be nice to have a tag that could be associated with the struct fileld to contain the human readable column header?